### PR TITLE
ci(blobstorage): deploy blobs swagger api

### PR DESCRIPTION
**Pull request by RogerLamTd**
https://blobstorage-swagger.vercel.app/

### CODE REVIEW

File: .github/workflows/blobstorage-preview.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+env:
+  VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+  VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID_BLOBS }}
```
Comment (d1onys1us) [inline .github/workflows/blobstorage-preview.yml:3]: better to follow the same naming convention. i think that would be `VERCEL_PROJECT_ID_BLOBSTORAGE`

File: packages/blobstorage/.gitignore [hunk 2]
```diff
@@ -19,3 +19,4 @@
 
 # Go workspace file
 go.work
+.vercel
```
Comment (d1onys1us) [inline packages/blobstorage/.gitignore:22]: we can just add this to the root gitignore because other vercel projects will benefit from it.

**Comment by d1onys1us** [comment]
@RogerLamTd we need one for `blobstorage-production.yml` as well i believe. it will be very similar, you can copy the other examples.

**Review by dionysuzx — APPROVED** [review]

**Comment by d1onys1us** [comment]
LGTM

**Review by dantaik — APPROVED** [review]

**Pull request merged**